### DISCUSSION TITLE
How to install Starship with no root access?

### DISCUSSION THREAD

**Discussion by ghost** [starship/starship#3843] [Q&A] [answered]
Hi,

As i don't have access to /usr/local/bin ... Is there a way to install starship without root access?

Thanks :D

**Comment by davidkna** [comment, marked as answer]
The installation script has an option to install the binary in a different location. If that location is writable, the script won't request root access. For instance, to install starship to `$HOME/.local/bin` you could use this:
```sh
mkdir -p "$HOME/.local/bin" && curl -sS https://starship.rs/install.sh | sh -s - --bin-dir "$HOME/.local/bin"
```

**Reply by ghost** [reply]
it worked, thank you very much :D
